README.md

```diff
@@ -1,2 +1,6 @@
 # city-sr-api-example
 Example code and documentation for talking to the City's Service Request API
+![image](Screenshot1.png)
+![image](Screenshot2.png)
+![image](Screenshot3.png)
+![image](Screenshot4.png)
```

Comment:
I'm not wild about images of code in documentation - readers can't copy them out, and use them in their own code. 

Why don't you just put the relevant YAML in formatted code blocks?

e.g.
```yaml
my key: "my value"
my other array:
  - val 1
  - val 2
  - etc
```